localstack/services/awslambda/lambda_executors.py

```diff
@@ -1410,18 +1477,6 @@ def docker_image_for_lambda(cls, lambda_function: LambdaFunction):
             )
         docker_tag = runtime
         docker_image = config.LAMBDA_CONTAINER_REGISTRY
-        # TODO: remove prefix once execution issues are fixed with dotnetcore/python lambdas
```

Comment:
These patches seem to be no longer required - in fact, some of these older versions do not support `$DOCKER_LAMBDA_STAY_OPEN`, hence removing this code from here.